sql/load_demo_data.sql

```diff
@@ -6251,6 +6251,7 @@ CREATE TABLE `oh_user` (
   `US_ID_A` varchar(50) NOT NULL DEFAULT '',
   `US_UG_ID_A` varchar(50) NOT NULL DEFAULT '',
   `US_PASSWD` varchar(60) NOT NULL DEFAULT '',
+  `US_PASSWD` varchar(60) NOT NULL DEFAULT '',
```

Comment:
is this adding real and so a mistake or it's just Github showing it?

![image](https://github.com/informatici/openhospital-core/assets/2938553/23569d5e-a011-49e2-adc5-52b74aea3c4f)